src/scripts/stars.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+let starsAll = function () {
+  let stars = document.querySelectorAll('.stars');
+
+  function counterStars (event) {
+    let childStar = event.target;
+    let parentStar = childStar.parentNode;
+    let childrenStar = parentStar.children;
+    let counterStar = 0;
+
+    if (childStar.classList.contains('star')) {
+      // Counting nr of stars
+      while ((childStar = childStar.previousElementSibling) != null) {
+        counterStar++;
+        for (let x = 0; x <= counterStar; x++) {
+          childrenStar[x].classList.add('rated-star');
+          childrenStar[x].style.color = '#d58e32';
+        }
+      }
+      if (event.target === parentStar.firstElementChild) {
```

Comment:
ten event.target masz przypisany do zmiennej

Reply:
testowalam to wczesniej, zmienna do ktorej dodany jest event.target przechodzi wyzej w linii 14 przez petle while, dlatego w tym przypadku nie moge jej uzyc. Wywala bledy.